src/types.ts

```diff
@@ -94,6 +94,8 @@ export interface IWsdlBaseOptions {
   wsdl_options?: { [key: string]: any };
   /** set proper headers for SOAP v1.2. */
   forceSoap12Headers?: boolean;
+  /** set content type header action for SOAP v1.2 */
+  addHeadersAction?: boolean;
```

Comment:
@gbar can you follow up with a PR to add this to the README please?

Reply:
Will add that when I get home. Will create another PR with that